rules.adoc

```diff
@@ -199,9 +199,19 @@ imparts dynamic back spin on the ball to keep the ball on its surface.
 
 Other players must be able to access the ball.
 
-{++The ball needs to stay within the bounds of the field, as defined by the
+The ball needs to stay within the bounds of the field, as defined by the
 walls. If a robot moves the ball outside of the field (that is, beyond the walls
-or above their height), it is deemed damaged. (<<damaged-robots>>)++}
+or above their height), it is deemed damaged. (<<damaged-robots>>)
+
+{~~A robot must touch the ball that is placed no further than 20 cm from any point
+on its convex hull within 10 seconds. If a robot does not do so within the time
+limit, it is deemed to be damaged. ~>
+Any robot must approach and touch the ball when it is placed on the nearest
+neutral spot before lack of progress is called. When on its own side of the field,
+any robot must be able to move the ball from the nearest neutral spot to the
+opponent's side of the field. If a robot does not do these things, it is deemed to be damaged.
```

Comment:
Which robot is deemed to be damaged? All on the field? The nearest one?

Reply:
That's going to be the one that is not able to play the ball on the nearest neutral spot. 🙂